.github/workflows/pr-opened.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: PR opened
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  add-welcome-message:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
```

Comment:
Confusing but I think indeed the issue api is the same as for a PR. Did you test it? Maybe good to add a small comment that the API is the same. And the intention is a comment on the PR

Reply:
Sure, [tests done](https://github.com/Hapag-Lloyd/test/pull/4).

Yeah, the same API. issues <==> PRs